src/boot/bootctl.c

```diff
@@ -251,6 +259,30 @@ static int get_file_version(int fd, char **v) {
         return r;
 }
 
+static const char *efi_arch(void) {
+#ifdef __x86_64__
+        _cleanup_free_ char *platform_size = NULL;
+        int r;
+
+        r = read_one_line_file("/sys/firmware/efi/fw_platform_size", &platform_size);
+        if (r == -ENOENT)
+                return EFI_MACHINE_TYPE_NAME;
+        if (r < 0) {
+                log_error_errno(r, "Error reading EFI firmware platform size: %m");
```

Comment:
given this is not fatal for anything, it's mostly a warning and thus should be downgraded to log_warning_errno()...

So a "platform size" is a weird concept. I figure they mean "word size"? maybe write that in the log message, i.e.

"Error reading EFI firmware platform word size, using the default word size"